docker-compose.yml

```diff
@@ -67,8 +67,10 @@ services:
       << : *py-environment
       PORT: 8011
     env_file: .env
-    links:
+    depends_on:
```

Comment:
This change is not in the initial upgrade PR. Is this by accident?

Reply:
Oh yes, thank you for catching that and picking up the PR quickly!